enhancements/network-observability/network-observability.md

```diff
@@ -161,6 +174,40 @@ prior to enabling network observability, will require additional resources.
 The specific number of cores and the amount of memory and storage required
 will be finalized as more testing is done.
 
+### CRD
```

Comment:
Hey @JoelSpeed ,
This is the PR I talked about in slack with you, for an operator that is going to be productized.
There's this section about the CRD you may want to review (or someone from your team). The full CRD doc is [in a separate file](https://github.com/netobserv/network-observability-operator/blob/main/docs/FlowCollector.md), let us know if you'd like us to copy it entirely here for an easier review.

Reply:
This is the file we would normally review, https://github.com/netobserv/network-observability-operator/blob/main/api/v1alpha1/flowcollector_types.go

My suggestion would be to open a PR to openshift/api that you don't intend to merge (unless you want to locate your APIs there for separation), and then the API review team can review in normal GitHub fashion, then you can take that feedback into your next version/your stable version

Reply:
here we go: https://github.com/openshift/api/pull/1218

Reply:
I've left some high level feedback, let me know if you want to discuss any of these further